skimage/filters/ridges.py

```diff
@@ -438,9 +440,14 @@ def frangi(image, sigmas=range(1, 10, 2), scale_range=None, scale_step=None,
 
         # Compute output image for given (sigma) scale and store results in
         # (n+1)D matrices, see equations (13) and (15) in reference [1]_
-        filtered_array[i] = ((1 - np.exp(-r_a / alpha_sq))
-                             * np.exp(-r_b / beta_sq)
+        filtered_array[i] = (np.exp(-r_b / beta_sq)
                              * (1 - np.exp(-r_g / gamma_sq)))
+        if ndim == 3:
```

Comment:
Why did you add this part? Was the behaviour not correct for 3D images? If this is the case, could you please also add a test here?

Reply:
The 3D image case is managed [here](https://github.com/scikit-image/scikit-image/pull/4343/files#diff-9a205ce8d1dfde393db115dfeb32740bL427) in the previous implementation. It is working perfectly well, I just rewrite things to be sure that I correctly understood the behavior :slightly_smiling_face: I will revert it, sorry :pray: